packages/frontend-shared/cypress/support/component.ts

```diff
@@ -20,6 +21,7 @@ cy.gqlStub = GQLStubRegistry
 
 Cypress.on('uncaught:exception', (err) => !err.message.includes('ResizeObserver loop limit exceeded'))
 
+configure({ testIdAttribute: 'data-cy' })
```

Comment:
What about putting this in `packages/frontend-shared/cypress/support/common.ts`? We also do the same `configure` call [here](https://github.com/cypress-io/cypress/blob/develop/packages/app/cypress/component/support/index.ts#L30). That same file also [imports common](https://github.com/cypress-io/cypress/blob/develop/packages/app/cypress/component/support/index.ts#L1), so if we do `config` in `common`, then we could also remove it from `app` and just have it in one place.

Reply:
I like!